services/client/src/styles/_variables.scss

```
@@ -1,41 +1,77 @@
 $color-primary: #0b6394;
 $color-primary-dark: #084b6f;
+$color-primary-light: #2a86bd;
+$color-primary-tint: #e8f3fb;
+
+// Login colors
+$color-ink: #0a2942;
+$color-ink-deep: #061c30;
+$color-parchment: #f3ecdd;
+$color-parchment-line: #d9cdb4;
+
 $color-success: #2e7d32;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)